arb_os/arbsys.mini

```diff
@@ -65,12 +69,14 @@ import impure func decompressAddress(stream: ByteStream) -> option<(ByteStream,
 import func rlp_encodeUint(val: uint, ba: ByteArray, offset: uint) -> (ByteArray, uint);
 import func rlp_encodeAddress(addr: address, ba: ByteArray, offset: uint) -> (ByteArray, uint);
 
+import impure func pcTableForCode(code: ByteArray) -> option<Stack>;
+
 
 public impure func arbsys_txcall() {
     if let Some(topFrame) = evmCallStack_topFrame() {
         let calldata = evmCallFrame_getCalldata(topFrame);
         if (bytearray_size(calldata) < 4) {
-            evmOp_revert(0, 0);
+            evmOp_revert_knownCodePc(address(100), 0, 0, 0);
```

Comment:
It looks like `evmOp_revert_knownCodePc(address(100), 0, 0, 0);` is still used a few times in this file. I'm not sure if that was intentional after the last commit replaced most of them

Reply:
Fixed.